api/handlers/containerhandler.py

```diff
@@ -0,0 +1,352 @@
+import datetime
+import logging
+
+import json
+import bson
+import copy
+import os
+
+from ..dao import APIStorageException, containerstorage
+from ..auth import containerauth, always_ok
+from .. import validators
+from .. import debuginfo
+from .. import files
+from .. import base
+from .. import util
+
+
+log = logging.getLogger('scitran.api')
+
+
+class ContainerHandler(base.RequestHandler):
+    """
+    This class handle operations on a generic container
+
+    The pattern used is:
+    1) load the storage class used to interact with mongo
+    2) configure the permissions checker and the json payload validators
+    3) validate the input payload
+    4) augment the payload when appropriate
+    5) exec the request (using the mongo validator and the permissions checker)
+    6) check the result
+    7) augment the result when needed
+    8) return the result
+
+    Specific behaviors (permissions checking logic for authenticated and not superuser users, storage interaction)
+    are specified in the container_handler_configurations
+    """
+    use_oid = {
```

Comment:
it means object id.
groups use simple strings for ids,
projects, sessions and acquisitions instead use them.

I will add a comment ;)